src/handlers/rentals.ts

```diff
@@ -9,8 +9,12 @@ import { OperatorUpdated, AssetRented, AssetClaimed } from '../entities/Rentals/
 export function handleAssetRented(event: AssetRented): void {
   let contractAddress = event.params._contractAddress.toHexString()
   let tokenId = event.params._tokenId
+
+  let currentRentalIndex = getRentalsCount(contractAddress, tokenId)
   let rentalIndex = getRentalsNextCount(contractAddress, tokenId)
```

Comment:
maybe rename to newRentalIndex

Reply:
you mean `rentalIndex` right? good point!